src/components/sections/FindPokemon/FindPokemon.tsx

```diff
@@ -0,0 +1,33 @@
+
+import useForm from "../../../hooks/useForm";
+import InputSearch from "../../InputSearch/InputSearch";
+import styles from "./FindPokemon.module.scss";
+
+interface FormInput {
+  name: string,
+}
+
+const initialState: FormInput = {
+  name : '',
```

Comment:
Keep consistency with quotes. It seems you have been using double quotes everywhere, so use double quotes here as well